src/pattern-library/components/patterns/UsingComponentsPage.tsx

```diff
@@ -1,331 +1,320 @@
+          <Library.Link href="/input-button">
+            <code>Button</code>
+          </Library.Link>
+          ,{' '}
+          <Library.Link href="/navigation-link">
+            <code>Link</code>
+          </Library.Link>
         </p>
 
-        <Library.Pattern title="Props API">
+        <Library.Section id="presentational-components-api" title="API">
           <p>
-            <strong>Base components</strong> extend presentational component
-            props with an optional boolean <code>unstyled</code> prop. When set,
-            the component will have no styling at all, allowing for complete
-            customization.
+            All presentational components provide this API unless documented
+            otherwise.
           </p>
-        </Library.Pattern>
-      </Library.Section>
+          <Library.Example title="children">
+            <Library.Info>
+              <Library.InfoItem label="type">
+                <code>preact.ComponentChildren</code>
+              </Library.InfoItem>
+            </Library.Info>
+          </Library.Example>
+          <Library.Example title="classes">
+            <Library.Info>
+              <Library.InfoItem label="description">
+                CSS class(es) that will be appended to the CSS classes applied
+                to the {"component's"} outermost or most significant element.
+                This can be used independently or in conjunction with the
+                Styling API props.
+              </Library.InfoItem>
+              <Library.InfoItem label="type">
+                <code>string</code>
+              </Library.InfoItem>
+            </Library.Info>
+          </Library.Example>
+          <Library.Example title="elementRef">
+            <Library.Info>
+              <Library.InfoItem label="description">
+                A <code>Ref</code> applied to the {"component's"} outermost or
+                most significant element.
+              </Library.InfoItem>
+              <Library.InfoItem label="type">
+                <code>{'preact.Ref<HTMLElement | undefined>'}</code>
+              </Library.InfoItem>
+            </Library.Info>
+          </Library.Example>
+          <Library.Example title="...htmlAttributes">
+            <Library.Info>
+              <Library.InfoItem label="description">
+                Presentational components accept HTML attributes applicable to
+                the element type of the {"component's"} outermost or most
+                significant element.
+              </Library.InfoItem>
+              <Library.InfoItem label="example">
+                <Library.Code
+                  content={`<Link id="my-own-id" href="https://www.example.com">A Link with attributes</Link>`}
+                  size="sm"
+                />
+              </Library.InfoItem>
+              <Library.InfoItem label="type">
+                <code>
+                  {'preact.JSX.HTMLAttributes<T extends HTMLElement>'}
```

Comment:
I'm not at all convinced I got "types" right in these docs so I'm open to feedback.